web/handlers/invites.go

```diff
@@ -39,19 +40,31 @@ type inviteHandler struct {
 	deniedKeys    roomdb.DeniedKeysService
 }
 
-func (h inviteHandler) buildJoinRoomURI(token string) template.URL {
-	var joinRoomURI url.URL
-	joinRoomURI.Scheme = "ssb"
-	joinRoomURI.Opaque = "experimental"
-
+func (h inviteHandler) buildJoinRoomURI(token string, userAgent string) template.URL {
 	queryVals := make(url.Values)
 	queryVals.Set("action", "claim-http-invite")
 	queryVals.Set("invite", token)
 
 	submissionURL := h.urlTo(router.CompleteInviteConsume)
 	queryVals.Set("postTo", submissionURL.String())
 
-	joinRoomURI.RawQuery = queryVals.Encode()
+	joinRoomURI := url.URL{
+		Scheme:   "ssb",
+		Opaque:   "experimental",
+		RawQuery: queryVals.Encode(),
+	}
+
+	// Special treatment for Android Chrome for issue #135
+	// https://github.com/ssb-ngi-pointer/go-ssb-room/issues/135
+	browser := ua.Parse(userAgent)
+	if browser.IsAndroid() && browser.IsChrome() {
+		joinRoomURI = url.URL{
+			Scheme:   "intent",
+			Opaque:   "//experimental",
+			RawQuery: queryVals.Encode(),
+			Fragment: "Intent;scheme=ssb;end;",
+		}
+	}
```

Comment:
I'd prefer to have this and the one in aliases in one place but I wouldn't call veto for it.

`web/utils.go` is a bit of a grab bag of things right now. Maybe it could be a function, there?

Reply:
Thanks! I also thought it should be centralized somewhere but was waiting for a good suggestion. I think I'll do that